tests/test_activate.py

```diff
@@ -2474,6 +2461,9 @@ def clear(self) -> None:
         self.sendline(f"echo {marker}")
         self.expect(rf"{marker}\r?\n")
 
+    def path_conversion(self, *args, **kwargs):
+        return self.activator.path_conversion(*args, **kwargs)
```

Comment:
Is this new API really needed?

Reply:
🤷🏻‍♂️ I'm indifferent about this, just made the usage elsewhere a tad shorter